src/boot/boot.janet

```diff
@@ -2903,14 +2910,17 @@
     (do
       (def x (dyn sym))
       (if (not x)
-        (do
-          (def [fullpath mod-kind] (module/find (string sym)))
-          (if-let [mod-env (in module/cache fullpath)]
-            (print-module-entry {:module     true
-                                 :kind       mod-kind
-                                 :source-map [fullpath nil nil]
-                                 :doc        (in mod-env :doc)})
-            (print "symbol " sym " not found.")))
+        (if ({'break 1 'def 1 'do 1 'fn 1 'if 1 'quasiquote 1 'quote 1
```

Comment:
`index-of` will be a little more concise for this sort of operation.

Reply:
Nice, thanks.